spec/requests/api/v2.0/sources_spec.rb

```diff
@@ -134,16 +134,6 @@
           :parsed_body => Insights::API::Common::ErrorDocument.new.add(400, "Record not unique").to_h
         )
       end
-
-      it "ignores blacklisted params" do
```

Comment:
Is this still needed to test https://github.com/RedHatInsights/sources-api/blob/master/app/controllers/application_controller.rb#L92?

Or, does it fail on the post because "tenant" was removed from the schema?

Reply:
Right this fails sooner because tenant isn't even defined as an attribute.

This test still exists in the v1.0 spec